linode/manager
# chore: Update Vite, Storybook, tsup, and related dependencies

## Description 📝

-  Updates Vite, Storybook, tsup, and related dependencies
- I'm doing this because I'm hoping it _may_ improve the Storybook experience, overall performance, and move us off of beta versions. 

## Major Changes 🔄

- Updates Vite to latest
  -  The Vite team focused solely on performance for 4.3.0 so things may feel faster
- Updates Storybook to latest
- Updates tsup to latest

## How to test 🧪
- Pull down this PR
- Run `yarn` to install dependencies for good measure (optional because `yarn dev` does this too)
- Test Cloud Manager's local development server
  - `yarn dev`
- Test the Storybook local development server
  - `yarn storybook`
- Check Github Actions results for failures
